src/backend_skia_bindings/filter.rs

```diff
@@ -337,6 +337,7 @@ impl Filter<Surface> for SkiaFilter {
 
                     let a = calc(c1.a, c2.a, 1.0);
                     if a.is_fuzzy_zero() {
+                        i += 1;
```

Comment:
I think this is the only backend we use, it’s easier to maintain with a smaller diff

Reply:
That's true. Reverted the rest